lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -1,14 +1,28 @@
 ;(function (define) {
     'use strict';
 
-    define(['backbone', 'underscore', 'text!teams/templates/teams-tab.underscore'],
-        function (Backbone, _, teamsTabTemplate) {
-            var TeamTabView = Backbone.View.extend({
-                render: function() {
-                    this.$el.html(_.template(teamsTabTemplate, {}));
-                }
-            });
+    define(['backbone',
+            'underscore',
+            'gettext',
+            'js/components/header/views/header',
+            'js/components/header/models/header',
+            'text!teams/templates/teams-tab.underscore'],
+           function (Backbone, _, gettext, HeaderView, HeaderModel, teamsTabTemplate) {
+               var headerModel = new HeaderModel({
+                   description: gettext("Course teams are organized into topics created by course instructors. Try to join others in an existing team before you decide to create a new team!"),
+                   title: gettext("Teams")
+               });
+               var headerView = new HeaderView({
+                   model: headerModel
+               });
+               var TeamTabView = Backbone.View.extend({
+                   render: function() {
+                       this.$el.html(_.template(teamsTabTemplate, {}));
+                       this.$el.prepend(headerView.$el);
```

Comment:
I didn't realize that `headerView` gets a `$el` before its render method is called. In general I've seen the reverse; render the view, then append its `$el`. @andy-armstrong - is there a "right" way to do this?